internal/httperr/errors_test.go

```
@@ -161,6 +163,48 @@ func TestErrorHandler(t *testing.T) {
 		assert.Contains(t, rec.Body.String(), `"message":"resource not found"`)
 	})
 
+	t.Run("classifies an unmatched route as a correctable request", func(t *testing.T) {
+		router := echo.New()
+		router.HTTPErrorHandler = ErrorHandler
+		router.GET("/known", func(c echo.Context) error {
+			return c.NoContent(http.StatusNoContent)
+		})
+		req := httptest.NewRequest(http.MethodGet, "/missing", nil)
```

golangci-lint (2.13.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)